### ISSUE TITLE
Player Character Movement Limitations

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#170]
```
What steps will reproduce the problem?
1. Playing the game (in landscape mode) on a device that has a dedicated D-pad 
with "Straight (Original)" as the chosen option under "Preferences", then 
"Movement Method".

What is the expected output? What do you see instead?
To still be able to access all areas of a created map.
Restricted and limited movement. 

What version of the product are you using? On what device?
Version 0.6.8 on a Motorola Droid.

Please provide any additional information below.
A quick and easy example to duplicate my report is to attempt to access the 
small grove area located on the west side of Flagstaff where the beetles roam. 
Under the previously described setting, one cannot enter this area. However, if 
you play by touch screen, or by using the "Directional" setting under "Movement 
Method" found under "Preferences", you can easily access map areas that contain 
significant content as well as enemies located behind a tree, etc. I feel that 
the best and most efficient course of action would be to simply make 
"Directional" the default "Movement Method" and remove "Straight (Original)" as 
an option to eliminate this issue.

```

Original issue reported on code.google.com by `ctnbeh13@gmail.com` on 21 Mar 2011 at 11:35

**Comment by GoogleCodeExporter** [comment]
```
Yes, this is a bug in the map design of flagstone0.tmx. 
Thank you for reporting it.

The maps should allow you to reach all areas with only horizontal and vertical 
movement. Diagonal movement should not be required.

A fix for flagstone0.tmx will be included in the next update.

Thank you again for reporting it. Please report more such areas that you 
encounter. Hopefully there should be none.
```

Original comment by `oskar.wi...@gmail.com` on 22 Mar 2011 at 6:21
- Changed state: **Started**
- Added labels: **Priority-Medium**, **Milestone-0.6.9**

**Comment by GoogleCodeExporter** [comment]
Original comment by `oskar.wi...@gmail.com` on 2 Apr 2011 at 6:13
- Changed state: **Ready**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.9 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 29 Jun 2011 at 7:51
- Changed state: **Done**
